custom/icds_reports/ucr/data_sources/awc_mgt_forms.json

```diff
@@ -66,6 +66,28 @@
         },
         "column_id": "awc_id"
       },
+      {
+        "display_name": "",
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "expression": {
+          "datatype": null,
+          "type": "ancestor_location",
+          "location_id": {
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "awc_selected_location_id"
+            ]
+          },
+          "location_type": "supervisor",
+          "location_property": "_id"
+        },
+        "column_id": "supervisor_id"
```

Comment:
based on the other location expressions in this UCR I think this should make use of the `user_location_id` named expression rather than using the `awc_selected_location_id` property.

It would also be worth changing the other expressions (`state_id` and `block_id`) to use `ancestor_location` so that they can take advantage of the caching.

Reply:
Makes sense, will do that.

Reply:
I actually have an [old PR](https://github.com/dimagi/commcare-hq/pull/20142) that did this for all UCRs, I will see if I can revive that first.

Reply:
Revived the other one that updates this UCR as well

Reply:
Updated.